rhombus/private/annotation.rkt

```diff
@@ -791,6 +792,7 @@
 (define-annotation-syntax NonnegReal (identifier-annotation #'nonnegative-real? #'()))
 (define-annotation-syntax Void (identifier-annotation #'void? #'()))
 (define-annotation-syntax False (identifier-annotation #'not #'()))
+(define-annotation-syntax True (identifier-annotation #'(lambda (x) x) #'()))
```

Comment:
I think using `(lambda (x) (and x #t))` is slightly better so that it’s really a predicate.

Reply:
HF